app/assets/stylesheets/application.scss

```diff
@@ -411,6 +406,11 @@ body .video_replace {
   overflow: hidden;
 }
 
+.required label::after {
+  color: #ff0000;
```

Comment:
Can you copy to a variable? Or use an existing variable if there's an appropriate one.